exo1/lib/screens/users/add_user_toolbar.dart

```diff
@@ -1,15 +1,40 @@
+import 'package:exo1/domain/user.dart';
+import 'package:exo1/domain/user_repository.dart';
 import 'package:flutter/material.dart';
 
-class AddUserToolbar extends StatelessWidget {
+class AddUserToolbar extends StatefulWidget {
+  @override
+  _AddUserToolbarState createState() => _AddUserToolbarState();
+}
+
+class _AddUserToolbarState extends State<AddUserToolbar> {
+  var newUser = '';
+  TextEditingController _controller = TextEditingController();
```

Comment:
it is a good idea to always mark fields that should not change with the `final` modifier.

Dart also allows you to omit the declaration type when a variable is declared and initialized at the same time, so this can be written `final TextEditingController _controller = TextEditingController()` or `final _controller = TextEditingController()`, as preferred